crates/uv/src/commands/tool/mod.rs

```diff
@@ -164,7 +170,13 @@ impl<'a> Target<'a> {
     /// Returns whether the target package is Python.
     pub(crate) fn is_python(&self) -> bool {
         let name = match self {
-            Self::Unspecified(name) => name,
+            Self::Unspecified(name) => {
+                // Identify the package name from the PEP 508 specifier.
+                let index = name
+                    .find(|c| !matches!(c, 'A'..='Z' | 'a'..='z' | '0'..='9' | '-' | '_' | '.'))
+                    .unwrap_or(name.len());
+                &name[..index]
```

Comment:
I'm considering refactoring this whole enum, but this seems fine for now.

Reply:
For context, this is like, "Given `ruff<0.5.0`, find `ruff`", i.e., the command name.